components/Common/CrossLink/index.tsx

```diff
@@ -0,0 +1,40 @@
+import styles from './index.module.scss';
+import Link from 'next/link';
+import classNames from 'classnames';
+import { ChevronLeftIcon, ChevronRightIcon } from '@heroicons/react/24/solid';
+import { FormattedMessage } from 'react-intl';
+import type { FC } from 'react';
+
+type CrossLinkProps = {
+  direction: 'left' | 'right';
+  text: string;
+  url: string;
+};
+
+const CrossLink: FC<CrossLinkProps> = ({ direction, text, url }) => {
+  const contentClasses = classNames(styles['content'], {
+    [styles['right']]: direction === 'right',
+  });
+
+  return (
+    <Link className={styles.crossLink} href={url}>
+      {direction === 'left' && (
+        <div className={styles.header}>
+          <ChevronLeftIcon className={styles.icon} />
+          <FormattedMessage id="components.common.crossLink.prev" />
+        </div>
+      )}
+
+      {direction === 'right' && (
+        <div className={`${styles.header} ${styles.right}`}>
+          <FormattedMessage id="components.common.crossLink.next" />
+          <ChevronRightIcon className={styles.icon} />
+        </div>
+      )}
+
+      <div className={contentClasses}>{text}</div>
+    </Link>
+  );
```

Comment:
```suggestion
  return (
    <Link className={styles.crossLink} href={url}>
      <div className={`${styles.header} ${styles[direction]}`}>
        {direction === 'right' ? (
          <>
            <ChevronRightIcon className={styles.icon} />
            <FormattedMessage id="components.common.crossLink.next" />
          </>
        ) : (
          <>
            <FormattedMessage id="components.common.crossLink.prev" />
            <ChevronLeftIcon className={styles.icon} />
          </>
        )}
      </div>
      <div className={contentClasses}>{text}</div>
    </Link>
  );
```
this is what I would have done. What do you think?

Reply:
I think it's better to be explicit (writing both `left` and `right` conditionals). Using ternary operator might be more efficient but it's less readable IMO.

Reply:
Voting for original variant, I don't see a need of ternary here

Reply:
@AugustinMauroy I'll stick to the current variant since there's no strong reason to change, resolving this comment. Feel free to unresolve if there are more things to discuss. 😄

Reply:
I was just suggesting a variant. But it's true that using the ternary operator isn't very clever because it suggests that it's true false when in our case it's "left" "right". Super discution Thanks to you (2)